src/sys/time.rs

```diff
@@ -144,6 +144,20 @@ impl TimeValLike for TimeSpec {
 }
 
 impl TimeSpec {
+    /// Makes a new `TimeSpec` with the special UTIME_NOW constant
+    #[inline]
+    #[cfg(target_os = "linux")]
```

Comment:
Plenty of OSes besides Linux have `UTIME_NOW` and `UTIME_OMIT`.  Can you please enable all of them here?  You may need to make a PR to libc first.

Reply:
Yes, I'm happy to do that. I just don't know how to get that exhaustive list of OSes. Any suggestions?

Reply:
There's no easy way.  All of the open source OSes have online manpages; that's what I use.  The hard one is OSX.  I haven't found a good, authoritative source for OSX man pages.

Reply:
I think you're still leaving out some OS support.  FreeBSD, for one, defines these symbols.  I haven't checked others.  Can you please check their online man pages, and open a PR to libc to add support where it's missing?

Reply:
I've got a PR open here: https://github.com/rust-lang/libc/pull/1474